prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!-- eslint-disable prefer-const -->
+<!-- eslint-disable prefer-const -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'Jeremy Nguyen';
+      console.log(fullName);
+      console.log(typeof fullName);
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      const nothing = null;
+      // eslint-disable-next-line prefer-const
+      let never = console.log(nothing);
```

Comment:
this should be 
let never;

Reply:
<img width="305" alt="Screenshot 2023-06-20 at 8 15 47 PM" src="https://github.com/jeremyhnguyen/javascript-prep-0623-code-solutions/assets/136545701/c4e416bc-1cb2-45ee-b069-49c394fbb982">